pkg/langruntime/langruntime.go

```diff
@@ -165,12 +165,12 @@ func GetFunctionImage(runtime, ftype string) (string, error) {
 }
 
 // GetBuildContainer returns a Container definition based on a runtime
-func GetBuildContainer(runtime string, env []v1.EnvVar, runtimeVolume, depsVolume v1.VolumeMount) (v1.Container, error) {
```

Comment:
it is not required to have 2 volumes since `runtimeVolume` now is already populated with the content of depsVolume (done in the provision init container)